File: beecrowd_exercicios_em_python/begginner/ufpr_gaming.py
while True:
    try:
        n, i = map(int, input().split())

        vzs = 0

        for j in range(n):
            p, q = map(int, input().split())
            if p == i and q == 0:
                vzs += 1

        print(vzs)

    except EOFError:
        break

File: beecrowd_exercicios_em_Ruby/begginer/ufpr_gaming.rb
while (n = gets)

  i, j = n.split(' ').map(&:to_i)

  vzs = 0

  i.times do

    k, l = gets.split.map(&:to_i)
    vzs += 1 if k == j && l.zero?

  end

  puts vzs

end
